src/network/mcpe/NetworkSession.php

```diff
@@ -949,11 +950,12 @@ public function syncAvailableCommands() : void{
 
 	public function onChatMessage(Translatable|string $message) : void{
 		if($message instanceof Translatable){
-			$language = $this->player->getLanguage();
-			if(!$this->server->isLanguageForced()){
+			$language = Server::getInstance()->getLanguage();
+			$namespace = explode('.', $message->getText())[0] ?? '';
+			if(!in_array($namespace, $language->getNamespaces(), true) || !$this->server->isLanguageForced()){
 				//we can't send nested translations to the client, so make sure they are always pre-translated by the server
 				$parameters = array_map(fn(string|Translatable $p) => $p instanceof Translatable ? $language->translate($p) : $p, $message->getParameters());
-				$this->sendDataPacket(TextPacket::translation($language->translateString($message->getText(), $parameters, "pocketmine."), $parameters));
+				$this->sendDataPacket(TextPacket::translation($language->translateString($message->getText(), $parameters), $parameters));
```

Comment:
This breaks unforced translations.

Reply:
`ServerLanguage::getNamespaces()` contains the pocketmine namespace meaning this will never affect the output of translateString

Reply:
I don't think you understand what the purpose of this is. The point is to allow the client to translate non-`pocketmine` strings by itself. By removing this, you're forcing ALL the translations to be translated by the server.